docs/topics/multiplatform/multiplatform-dsl-reference.md

```diff
@@ -147,17 +156,36 @@ kotlin {
 }
 ```
 
-### JavaScript targets
+### Web targets
 
-The `js` block describes the configuration of JavaScript targets. It can contain one of two blocks depending on the target execution environment:
+The `js` block describes the configuration of Kotlin/JS targets, and the `wasmJs` block describes the configuration of
+Kotlin/Wasm targets interoperable with JavaScript. They can contain one of two blocks depending on the target execution
+environment:
 
-| **Name**  | **Description**                      | 
-|-----------|--------------------------------------|
-| `browser` | Configuration of the browser target. |
-| `nodejs`  | Configuration of the Node.js target. |
+| **Name**              | **Description**                      | 
+|-----------------------|--------------------------------------|
+| [`browser`](#browser) | Configuration of the browser target. |
+| [`nodejs`](#node-js)  | Configuration of the Node.js target. |
 
 Learn more about [configuring Kotlin/JS projects](js-project-setup.md).
 
+A separate `wasmWasi` block describes the configuration of Kotlin/Wasm targets that support the WASI system interface.
+It can contain the following blocks:
+
+| **Name**             | **Description**                                                                                                                | 
+|----------------------|--------------------------------------------------------------------------------------------------------------------------------|
+| [`nodejs`](#node-js) | Configuration of the Node.js target.                                                                                           |
```

Comment:
`nodejs` is now the only option to be used with wasi
`binaries` are available for `js` and `wasmJs` as well, it regulates whether user needs compilation of final binary or not
So `binaries.executable` should be in final module of all targets `js`, `wasmJs` and `wasmWasi`
So I am not sure that we need to describe it specifically in `wasmWasi`

Reply:
Thank you! Added  a paragraph about `binaries.executable` for all 3 targets plus a link to a detailed description